### PR TITLE
Improve wording in README

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Improve wording in README.
Reorganize MAINTAINERS file to put our WG leader at the top:D
Now I'm in the tail:-(

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
README.md

### CODE REVIEW

**Comment by Andersson007** [comment]
@bmalynovytch @Jorge-Rodriguez , I asked gundalow and suggested better wording. If it's fine, feel free to merge. Thanks:)

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ We follow [Ansible Code of Conduct](https://docs.ansible.com/ansible/latest/comm
 
 The content of this collection are made by [people](CONTRIBUTORS) like you.
 
-Every contribution is very welcomed!
+All types of contributions are very welcome.
```
Comment (Jorge-Rodriguez) [inline README.md]:
```suggestion
Any kind of contribution is very welcome.
```

**Review by Jorge-Rodriguez — COMMENTED** [review]
This is my own personal take on the wording

**Comment by Andersson007** [comment]
> This is my own personal take on the wording

Looks good to me too, thanks! committed

**Comment by Andersson007** [comment]
s/merged/committed/g

**Pull request merged**